source/extensions/filters/http/cache/cache_filter.cc

```diff
@@ -417,24 +419,25 @@ void CacheFilter::injectValidationHeaders(Http::RequestHeaderMap& request_header
          "injectValidationHeaders precondition unsatisfied: the "
          "CacheFilter is not validating a cache lookup result");
 
-  const Http::HeaderEntry* etag_header = lookup_result_->headers_->getInline(etag_handle.handle());
+  const Http::HeaderEntry* etag_header =
+      lookup_result_->headers_->getInline(CacheCustomHeaders::get().etag.handle());
   const Http::HeaderEntry* last_modified_header =
-      lookup_result_->headers_->getInline(last_modified_handle.handle());
+      lookup_result_->headers_->getInline(CacheCustomHeaders::get().last_modified.handle());
```

Comment:
here's an API idea for CacheCustomHeaders to make this nicer at call-sites:
```
  static const Http::HeaderEntry* last_modified_header = CacheCustomHeaders::lastModified(
    *lookup_result_->headers_);
``` 

basically, sink the `::get()` call, the `getInline()` call, and the `.handle()` reference into each API so it's terser at call-sites.

I'm OK either way but wanted to know what you thought.

Reply:
I implemented this suggestion, and it's quite a bit nicer at the call sites. Let me know what you think of the changes.

Reply:
Oh I see I didn't collapse the {get,set}Inline portion at call sites... I'll think a bit about how that might work.

Reply:
My initial thought is that, because we use both `setInline` and `getInline` from `headers_`, it's a bit more straight forward to leave it as it is now, rather than passing `headers_` to the methods and needing to add a `setInline` equivalent to the API for all of them. Does that sound reasonable?